core/io/resource_format_binary.cpp

```diff
@@ -772,10 +772,12 @@ Error ResourceLoaderBinary::load() {
 			}
 
 			res = Ref<Resource>(r);
-			if (!path.is_empty() && cache_mode != ResourceFormatLoader::CACHE_MODE_IGNORE) {
-				r->set_path(path, cache_mode == ResourceFormatLoader::CACHE_MODE_REPLACE); //if got here because the resource with same path has different type, replace it
-			} else if (!path.is_resource_file()) {
-				r->set_path_cache(path);
+			if (!path.is_empty()) {
+				if (cache_mode != ResourceFormatLoader::CACHE_MODE_IGNORE) {
+					r->set_path(path, cache_mode == ResourceFormatLoader::CACHE_MODE_REPLACE); //if got here because the resource with same path has different type, replace it
```

Comment:
```suggestion
					r->set_path(path, cache_mode == ResourceFormatLoader::CACHE_MODE_REPLACE); // If got here because the resource with same path has different type, replace it.
```

Reply:
```suggestion
					r->set_path(path, cache_mode == ResourceFormatLoader::CACHE_MODE_REPLACE); // If we got here because the resource with same path has different type, replace it.
```

Reply:
Applied.

So, could we say that a PR is responsible for the code sytle of any line it touches, regardless how deep the change on it is? Would that be a good way of thinking about this?